components/SlateRenderer/SlateRenderer.tsx

```diff
@@ -48,7 +49,7 @@ function SlateRenderer({ nodes }: Props) {
 
     return (
         <div className={styles.renderer}>
-            <Renderer nodes={nodes}>
+            <Renderer nodes={nodes} defaultComponents>
```

Comment:
@e1himself i assume that we need to add default components?

Reply:
It's not necessary, as `defaultComponents` is true by default.